<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseSelect</class>
 <widget class="QDialog" name="CaseSelect">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Select a case</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TIAS - Proof of concept</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>V. 0.02</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="CaseSelector"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Case Details</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="Info_Created_3">
            <property name="text">
             <string>CaseID : </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="I_Case">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="CreatedLabel">
            <property name="text">
             <string>Created : </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="I_Created">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="LeadExaminerLabel">
            <property name="text">
             <string>Lead Examiner : </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="I_Examiner">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Info_Created_4">
            <property name="text">
             <string>Case Name : </string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="I_CaseName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="Info_Created_5">
            <property name="text">
             <string>Number Of Images : </string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="I_Image">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="Info_Created_6">
            <property name="text">
             <string>Case Description :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="I_Description">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Load">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>New Case...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
